templates/dotnet/src/Appwrite/Models/Exception.cs.twig

```diff
@@ -0,0 +1,21 @@
+using System;
+
+namespace Appwrite
+{
+    public class {{spec.title | caseUcfirst}}Exception : Exception
+    {
+        public int? Code;
+        public string Response = null;
```

Comment:
Is response String?

Reply:
It must be the JSON string in .NET - since we don't have Response Models yet and C# doesn't like unknown data models 👍🏻